concrete-core/src/backends/core/implementation/engines/plaintext_creation.rs

```diff
@@ -0,0 +1,60 @@
+use crate::backends::core::implementation::engines::CoreEngine;
+use crate::backends::core::implementation::entities::{Plaintext32, Plaintext64};
+use crate::backends::core::private::crypto::encoding::Plaintext as ImplPlaintext;
+use crate::specification::engines::{PlaintextCreationEngine, PlaintextCreationError};
+
+impl PlaintextCreationEngine<u32, Plaintext32> for CoreEngine {
+    /// # Example:
+    /// ```
+    /// use concrete_core::prelude::*;
+    /// # use std::error::Error;
+    /// #
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// let mut engine = CoreEngine::new()?;
+    /// // Here a hard-set encoding is applied (shift by 20 bits)
+    /// let input = 3_u32 << 20;
+    /// let plaintext: Plaintext32 = engine.create_plaintext(&input)?;
+    /// engine.destroy(plaintext)?;
```

Comment:
Hi @Anas-B! In the doc test I only included checks on the variables we can get from a given output object (for example the amount of plaintexts in a plaintext vector). Since Plaintext32 doesn't have anything I can check there is no `assert_eq!`.